std/conv.d

```diff
@@ -510,7 +510,9 @@ T toImpl(T, S)(S value)
             T opCast(U)() @safe pure { assert(false); }
         }
     }
-    to!(Test.T)(Test.S());
+    // CHECKME: std/conv.d(513): Warning: calling std.conv.to!(T).to!(S).to without side effects discards return value of type T, prepend a cast(void) if intentional
+    // to!(Test.T)(Test.S());
+    cast(void) to!(Test.T)(Test.S());
```

Comment:
I realize you're just pulling in an upstream commit, dd82045, but this warning doesn't trigger for me with ldc ltsmaster on Android/ARM or with ldc 1.6 on linux/x64, with this cast removed.  Seems like a pointless addition, rather this were out.

Reply:
I will retest/recheck if this change is still required, when compiling on DragonFly.

Will remove/revert if not. If it is, we might have to figure out why and/or find some other work around.